samples/shell/echo/echo.yaml

```diff
@@ -12,3 +14,6 @@ spec:
   image: sk8s/echo:v0001
   protocol: stdio
   input: greetings
+  env:
+  - name: FUNCTION_URI
+    value: /resources/script/echo.sh
```

Comment:
why was this added here?

the current Dockerfile for the echo sample is:
```
FROM sk8s/shell-function-invoker:0.0.1-SNAPSHOT
ADD echo.sh /functions/script.sh
ENV FUNCTION_URI /functions/script.sh
```

Reply:
Interesting. Artifact of previous rebase I assume

Reply:
This is weird though, seems to indicate that the image ENV has precedence over what we set programmatically